src/sensors/MAX31865.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+// Adapted from https://github.com/adafruit/Adafruit_MAX31865
+
+/***************************************************
+  This is a library for the Adafruit PT100/P1000 RTD Sensor w/MAX31865
+  Designed specifically to work with the Adafruit RTD Sensor
+  ----> https://www.adafruit.com/products/3328
+  This sensor uses SPI to communicate, 4 pins are required to
+  interface
+  Adafruit invests time and resources providing this open source code,
+  please support Adafruit and open-source hardware by purchasing
+  products from Adafruit!
+  Written by Limor Fried/Ladyada for Adafruit Industries.
+  BSD license, all text above must be included in any redistribution
+ ****************************************************/
+
+#include <IO/SPI.h>
+#include <cstdint>
+#define MAX31865_CONFIG_REG 0x00
```

Comment:
I think using a `constexpr` or `enum class` for configuration flags would be benefitial for clarity compared to macros.

Reply:
> I think using a `constexpr` or `enum class` for configuration flags would be benefitial compared to macros.
> eg: 
> 
> ```c++
> namespace MAX31865 {
> enum class Config {
>     Bias = 0x80,
>     ModeAuto = 0x40,
>     ...
> }
> ...
> }
> 
> // When referencing
> MAX31865::Config::Bias
> ```
> 
> We don't need that necessarily, but I think it would be benefitial for organization reasons and to reduce the `#define`, `#ifdef`, `#ifndef` hell we have

Good point. We should definitely try to move away from macros to variables. I'll do that change when I'll have time, maybe tomorrow.